images/kubeflow-jupyter-web-app/tests/01-runs.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+set +o pipefail  # We expect the command to fail, but want its output anyway.
+docker run -t $IMAGE_NAME 2>&1 | grep "Invalid kube-config file. No configuration found."
```

Comment:
I think we can skip this test, since it doesn't catch much, and we have a real test that does catch stuff.

Reply:
makes sense